pyproject.toml

```diff
@@ -2,7 +2,7 @@
 requires = [
     "setuptools",
     "wheel",
-    "mesa~=2.1.0",
+    "mesa~=2.2",
```

Comment:
Shouldn't this be 2.2.0, for 2.2.x?

Reply:
In that case we'll need to have a new release for mesa-geo when mesa releases 2.3. But I'm thinking to avoid this because mesa-geo doesn't have frequent updates as compared to mesa. So we'll end up only updating version numbers.

Besides, 2.3 should not have breaking changes so I think it should be fine.

Reply:
Then we need to communicate in the release note that the pinning is now on the major version instead of the minor version.

Reply:
Updated the release notes for this